simulator-samples/sample-swagger/src/test/resources/templates/order_invalid_date.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": citrus:randomNumber(10),
```

Comment:
this could not be quoted, right? because right now the JSON is invalid.

Reply:
See above. There is a delete test already.